catroid/src/main/java/org/catrobat/catroid/ui/ProjectUtils.kt

```diff
@@ -46,32 +46,46 @@ import org.catrobat.catroid.content.bricks.ForeverBrick
 import org.catrobat.catroid.content.bricks.IfLogicBeginBrick
 import org.catrobat.catroid.content.bricks.IfThenLogicBeginBrick
 import org.catrobat.catroid.content.bricks.LookRequestBrick
+import org.catrobat.catroid.content.bricks.OpenUrlBrick
 import org.catrobat.catroid.content.bricks.ParameterizedBrick
 import org.catrobat.catroid.content.bricks.PhiroIfLogicBeginBrick
 import org.catrobat.catroid.content.bricks.RepeatBrick
 import org.catrobat.catroid.content.bricks.RepeatUntilBrick
 import org.catrobat.catroid.content.bricks.StartListeningBrick
 import org.catrobat.catroid.content.bricks.WebRequestBrick
 
+enum class SuspiciousBricks {
+    NO_SUSPICIOUS_BRICKS, CONTAINS_WEB_ACCESS_BRICK, CONTAINS_START_LISTENING_AND_WEB_ACCESS_BRICKS
+}
 /**
  * extension boolean function for List<Brick> data type.
  * check if the list contains suspicious bricks
  * */
-private fun List<Brick>.containsSuspiciousBricks(): Boolean {
+private fun List<Brick>.containsStartListeningAndWebAccessBricks(): SuspiciousBricks {
+    val backgroundRequestOrWebRequestBrickExists = any { brick ->
+        brick is WebRequestBrick || brick is BackgroundRequestBrick || brick is LookRequestBrick || brick is OpenUrlBrick
+    }
     val startListeningBrickExists = any { brick ->
         brick is StartListeningBrick
     }
-    val backgroundRequestOrWebRequestBrickExists = any { brick ->
-        brick is WebRequestBrick || brick is BackgroundRequestBrick || brick is LookRequestBrick
+    return when {
```

Comment:
this code part leads to warnings in Jenkins, have a look at the results from the latest run